elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -93,18 +93,20 @@ class PfeJumpLinksNav extends PFElement {
         if (this.getAttribute("sr-text")) {
           this.shadowRoot.querySelector("nav").prepend(div);
         }
-
+        if (this.hasAttribute("horizontal")) {
+          return;
+        }
         let html = "";
-        if (this.querySelector(".pfe-jump-links-nav--heading")) {
-          html = this.querySelector(".pfe-jump-links-nav--heading").cloneNode(
-            true
-          );
+        if (this.has_slot("pfe-jump-links-nav--heading")) {
+          html = this.querySelector("[slot='pfe-jump-links-nav--heading']")
+            .innerHTML;
+        } else {
+          html = "Jump to section";
         }
         if (!this.hasAttribute("horizontal")) {
           if (html) {
```

Comment:
I don't see how this can ever be false since html is initialized on line 99-104 above